doc/headlines.txt

```diff
@@ -165,6 +174,9 @@ fat_headlines                                        *headlines-fat_headlines*
 ==============================================================================
  4. CHANGELOG                                            *headlines-changelog*
 
+2.1.2
```

Comment:
should be 2.2.0
new feature means minor patch. But you can remove this here, I'll make a new commit to update the version when I merge it into master.